Ms2Database/DBClasses/Account.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Ms2Database.DBClasses
+{
+    public class Account
+    {
+        public Account()
+        {
+            CreationTime = DateTime.Now;
+        }
+
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        [Required]
+        [Key]
+        public long AccountId { get; set; }
+
+        [Column(TypeName = "nvarchar")]
+        [StringLength(10)]
+        public string Username { get; set; }
+
+        [Column(TypeName = "nvarchar")]
+        [StringLength(12)]
```

Comment:
Username and password should have bigger string sizes, but for now it's good!

Reply:
I agree. I just put a decently small size for now .